synopsys-task/src/synopsys-task/application-constant.ts

```diff
@@ -214,8 +214,60 @@ export const EXIT_CODE_MAP = new Map<string, string>([
   ["3", "Failed to shutdown the bridge"],
   ["8", "The config option bridge.break has been set to true"],
   ["9", "Bridge initialization failed"],
+  ["101", "Requires at least one scan type"],
```

Comment:
Use constants instead of direct error codes so that later codes can be changed from one single source.
Example, REQUIRED_SCAN_TYPE_ERROR_CODE = 101
then,
`[errorCode.REQUIRED_SCAN_TYPE_ERROR, "Requires at least one scan type"]`

Follow this in all the places.

Reply:
these exit codes are related to bridge only, does bridge return such error and exit code? If not, then create another map to handle ADO related exit code and rename this as well with BRIDGE_EXIT_CODE_MAP.